pipeline_dp/dp_computations.py

```diff
@@ -38,13 +38,20 @@ def linf_sensitivity(self, metric):
         if metric == pipeline_dp.Metrics.MEAN:
             return self.max_contributions_per_partition * abs(
                 self.middle() - self.low)
+        if metric == pipeline_dp.Metrics.VAR:
+            return self.max_contributions_per_partition * abs(
+                self.middle_squares() - self.low ** 2)
```

Comment:
This is incorrect. self.low ** 2 is not the smallest squared value if self.low < 0.